packages/paste-design-tokens/tokens/aliases/line-height.yml

```diff
@@ -11,4 +11,6 @@ aliases:
   line-height-80: "2.5rem" # 28px * 1.3 = 40px / 16 = 2.5
   line-height-90: "2.75rem" # 32px * 1.3 = 44px / 16 = 2.75
   line-height-100: "3.25rem" # 40px * 1.3 = 52px / 16 = 3.25
+  line-height-105: "3.75rem" # TODO: * TODO: = 60px / 16 = 3.75
```

Comment:
Nit: remove todos

Reply:
working on the maths will remove todos when numbers are finalized